plugins/org.jboss.tools.jst.js.node/src/org/jboss/tools/jst/js/node/Activator.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.jst.js.node;
+
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.ui.plugin.AbstractUIPlugin;
+import org.jboss.tools.jst.js.node.preference.NodePreferenceHolder;
+import org.osgi.framework.BundleContext;
+
+/**
+ * @author "Ilya Buziuk (ibuziuk)"
+ */
+public class Activator extends AbstractUIPlugin {
```

Comment:
Not critical, but IMO it's better to give some more specific names to plugin activator classes than just Activator. It will be easier to search for the class if it's called Node(JS)Activator(Plugin) or something like that. It would also help you to see what activator it is w/o looking at this package name in expressions like NodePlugin.log() vs Activator.log().